app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUpdateTest.kt

```diff
@@ -209,6 +210,33 @@ class BloodSugarEntryUpdateTest {
         ))
   }
 
+  @Test
+  fun `when save is clicked when updating blood sugar and there are no validation errors, then update blood sugar`() {
```

Comment:
Reword this to `when save is clicked after the blood sugar is updated, without any validation errors, then update the blood sugar`?